### PR TITLE
[matlab/es-es] Translate MatLab to  es_ES

### PR DESCRIPTION
**Pull request by AlburIvan**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: es-es/matlab-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,568 @@
...
+% Para usar funciones o scripts, deben estar en su ruta o directorio actual
+path % muestra la ruta actual
+addpath /path/to/dir % agrega a la ruta
+rmpath /path/to/dir % elimina de la ruta
+cd /path/to/move/into % cambia de directorio
+
+
+% Las variables se pueden guardar en archivos .mat
+save('myFileName.mat') % Guarda las variables en su espacio de trabajo
+load('myFileName.mat') % Carga las variables guardadas en Workspace
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: en espacio de trabajo

File: es-es/matlab-es.html.markdown [hunk 2]
```diff
@@ -0,0 +1,568 @@
...
+
+% M-file Scripts
+% Un archivo de script es un archivo externo que contiene una secuencia de instrucciones.
+% Permiten evitar escribir repetidamente el mismo código en la ventana de comandos
+% Tienen extensiones .m
+
+% M-file Functions
+% Al igual que los scripts, y tienen la misma extensión .m
+% Pero puede aceptar argumentos de entrada y devolver una salida
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: pueden

File: es-es/matlab-es.html.markdown [hunk 3]
```diff
@@ -0,0 +1,568 @@
...
+
+% Detiene la ejecución del archivo y le da control al teclado: el usuario puede examinar
+% o cambiar las variables. Escriba 'return' para continuar la ejecución, o 'dbquit' para salir del teclado
+
+% Lectura de datos (también xlsread / importdata / imread para archivos de excel / CSV / image)
+fopen(filename)
+
+% Salida
+disp(a) % Imprime el valor de la variable a
+disp('Hello World') % Imprime una cadena
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: hola mundo

File: es-es/matlab-es.html.markdown [hunk 4]
```diff
@@ -0,0 +1,568 @@
...
+fprintf % Imprime en la ventana de comandos con más control
+
+% Declaraciones condicionales (los paréntesis son opcionales, pero buen estilo)
+if (a > 15)
+	disp('Greater than 15')
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: más de 15

File: es-es/matlab-es.html.markdown [hunk 5]
```diff
@@ -0,0 +1,568 @@
...
+elseif (a == 23)
+	disp('a is 23')
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: es 23

File: es-es/matlab-es.html.markdown [hunk 6]
```diff
@@ -0,0 +1,568 @@
...
+else
+	disp('neither condition met')
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: ¿ninguna condición ha cumplido?

File: es-es/matlab-es.html.markdown [hunk 7]
```diff
@@ -0,0 +1,568 @@
...
+zeros(m,n) % m x n matriz de 0
+ones(m,n) % m x n matriz de 1
+diag(A) % Extrae los elementos diagonales de una matriz A
+diag(x) % Construya una matriz con elementos diagonales enumerados en x, y ceros en otra parte
+eye(m,n) % Matriz de identidad
+linspace(x1, x2, n) % Devuelve n puntos equiespaciados, con min x1 y max x2
+inv(A) % Inverso de la matriz A
+det(A) % Determinante de A
+eig(A) % Valores propios y vectores propios de A
+trace(A) % Traza de la matriz: equivalente a la suma (diag (A))
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: equivalente a sum(diag(A))

File: es-es/matlab-es.html.markdown [hunk 8]
```diff
@@ -0,0 +1,568 @@
...
+any(A) % Determina si alguno de los elementos es distinto de cero o verdadero
+isequal(A, B) % Determina la igualdad de dos matrices
+numel(A) % Cantidad de elementos en matriz
+triu(x) % Devuelve la parte triangular superior de x
+tril(x) % Devuelve la parte triangular inferior de x
+cross(A,B) % Devuelve el producto cruzado de los vectores A y B
+dot(A,B) % Devuelve un producto escalar de dos vectores (debe tener la misma longitud)
+transpose(A) % Devuelve la transposición de A
+fliplr(A) % Voltee la matriz de izquierda a derecha
+flipud(A) % Voltear la matriz de arriba hacia abajo
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: ¿voltee?

File: es-es/matlab-es.html.markdown [hunk 9]
```diff
@@ -0,0 +1,568 @@
...
+[P, D] = eig(A) % eigen-decomposition: AP = PD, las columnas de P son autovectores y las diagonales de D'son valores propios
+[U,S,V] = svd(X) % SVD: XV = US, U y V son matrices unitarias, S tiene elementos diagonales no negativos en orden decreciente
+
+% Funciones comunes de vectores
+max     % componente más grande
+min     % componente más pequeño
+length  % longitud de un vector
+sort    % ordenar en orden ascendente
+sum     % suma de elementos
+prod    % product of elements
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: Producto de elementos

File: es-es/matlab-es.html.markdown [hunk 10]
```diff
@@ -0,0 +1,568 @@
...
+std     % desviación estándar
+perms(x) % enumera todas las permutaciones de elementos de x
+find(x) % Encuentra todos los elementos distintos de cero de x y devuelve sus índices, puede usar operadores de comparación,
+        % i.e. find( x == 3 ) devuelve índices de elementos que son iguales a 3
+        % i.e. find( x >= 3 ) devuelve índices de elementos mayores o iguales a 3
+
+
+% Clases
+% Matlab puede soportar programación orientada a objetos.
+% Las clases deben colocarse en un archivo del nombre de la clase con una extensión .m.
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: ¿con la extensión?

File: es-es/matlab-es.html.markdown [hunk 11]
```diff
@@ -0,0 +1,568 @@
...
+                        [o1.longitude]+[o2.longitude]);
+    end
+  end
+end
+% Fin WaypointClass.m
+
+% Podemos crear un objeto de la clase usando el constructor
+a = WaypointClass(45.0, 45.0)
+
+% Las propiedades de clase se comportan exactamente como Matlab Structures.
```
Comment (Menelion) [inline es-es/matlab-es.html.markdown]: como estructuras de Matlab

**Comment by Menelion** [comment]
@AlburIvan, any progress with this?

**Comment by AlburIvan** [comment]
@Menelion should be done now :)

**Comment by Menelion** [comment]
@AlburIvan ¡Gracias!

**Pull request merged**